modules/manage/pages/iceberg/redpanda-topics-iceberg-snowflake-catalog.adoc

```diff
@@ -0,0 +1,203 @@
+    CATALOG_URI = '<open-catalog-uri>'
+    WAREHOUSE = '<open-catalog-name>'
+  )
+  REST_AUTHENTICATION = (
+    TYPE = OAUTH
+    OAUTH_CLIENT_ID = '<open-catalog-connection-client-id>'
+    OAUTH_CLIENT_SECRET = '<open-catalog-connection-client-secret>'
+    OAUTH_ALLOWED_SCOPES = ('PRINCIPAL_ROLE:ALL')
+  )
+  REFRESH_INTERVAL_SECONDS = 30
+  ENABLED = TRUE;
+----
++
+Use your own values for the following placeholders:
++
+- `<catalog-integration-name>`: Provide a name for your Iceberg catalog integration in Snowflake. 
+- `<open-catalog-uri>`: Your https://docs.snowflake.com/en/sql-reference/sql/create-catalog-integration-open-catalog#required-parameters[Open Catalog account URI] (`https://<snowflake-orgname>-<account-name>.snowflakecomputing.com/polaris/api/catalog`).
+- `<open-catalog-name>`: The name of your catalog in Open Catalog.
+- `<open-catalog-connection-client-id>`: The client ID of the service connection you created in an earlier step.
+- `<open-catalog-connection-client-secret>`: The client secret of the service connection you created in an earlier step.
+
+=== Create Iceberg table in Snowflake
+
+After creating the catalog integration, you must create an externally-managed table in Snowflake. You'll run your Snowflake queries against this table.
+
+. In your Snowflake database, run the https://docs.snowflake.com/en/sql-reference/sql/create-iceberg-table-rest[CREATE ICEBERG TABLE] command. The following example also specifies that the table should automatically refresh metadata:
++
+[,sql]
+----
+CREATE ICEBERG TABLE <table-name>
+  CATALOG = '<catalog-integration-name>'
+  EXTERNAL_VOLUME = '<iceberg-external-volume-name>'
+  CATALOG_TABLE_NAME = '<topic-name>'
+  AUTO_REFRESH = TRUE
+----
++
+Use your own values for the following placeholders:
++
+- `<table-name>`: Provide a name for your table in Snowflake.
+- `<catalog-integration-name>`: The name of the catalog integration you configured in an earlier step.
+- `<iceberg-external-volume-name>`: The name of the external volume you configured using the Tiered Storage bucket.
+- `<topic-name>`: The name of the table in your catalog, which is the same as your Redpanda topic name.
+
+=== Query table
+
+To verify that Snowflake has successfully created the table containing the topic data, run the following:
+
+[,sql]
+----
+SELECT SYSTEM$LIST_ICEBERG_TABLES_FROM_CATALOG('<catalog-integration-name>');
+----
+
+[,sql]
+----
+SELECT * FROM <table-name>;
+----
+
+Your query results should look like the following:
+
+[,bash,.no-copy.no-placeholders]
+----
+# Example for redpanda.iceberg.mode=key_value
++-----------------------------------------------------------------------+
+| SYSTEM$LIST_ICEBERG_TABLES_FROM_CATALOG('<catalog_integration_name>') |
++-----------------------------------------------------------------------+
+| [{"namespace":"redpanda","name":"<table_name>"}]                      |
++-----------------------------------------------------------------------+
+----
```

Comment:
Seems like this and `SELECT SYSTEM$LIST_ICEBERG_TABLES_FROM_CATALOG('<catalog-integration-name>');` should go up right after creating the catalog integration